cdk/domino_cdk/provisioners/eks/eks_iam.py

```diff
@@ -83,10 +83,63 @@ def provision(self, stack_name: str, cluster_name: str, r53_zone_ids: List[str],
             ],
         )
 
+        # https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/helm-chart-aws-ebs-csi-driver-2.1.1/docs/example-iam-policy.json
+        ebs_policy_document = iam.PolicyDocument(
+            statements=[
+                iam.PolicyStatement(
+                    effect=iam.Effect.ALLOW,
+                    actions=[
+                        "ec2:CreateSnapshot",
+                        "ec2:AttachVolume",
+                        "ec2:DetachVolume",
+                        "ec2:ModifyVolume",
+                        "ec2:DescribeAvailabilityZones",
+                        "ec2:DescribeInstances",
+                        "ec2:DescribeSnapshots",
+                        "ec2:DescribeTags",
+                        "ec2:DescribeVolumes",
+                        "ec2:DescribeVolumesModifications",
+                    ],
+                    resources=["*"],
```

Comment:
Any chance to limit it, say by 

conditions={"StringNotEqualsIfExists": {"ecr:ResourceTag/domino-deploy-id": stack_name}}

Reply:
Trying with the correct condtion.
Look at what EKS Worker Nodes get.... the above is a bit pointless since it isn't being verified in some cases.

Reply:
This interferes with ASGs if you pick the wrong tag.

Reply:
verified a CDK deploy does work